docs/api/exchange_rates.rst

```diff
@@ -0,0 +1,465 @@
++----------------+-------------------------------------------------------------------------------+
+
+If you request for more detailed data, you will receive the defualt data with the following additional fields:
+
++----------------+-----------------------------------------+
+| Field          | Description                             |
++================+=========================================+
+| updatedAt      | Last update date of the exchange rate   |
++----------------+-----------------------------------------+
+| createdAt      | Creation date date of the exchange rate |
++----------------+-----------------------------------------+
+
+Creating an Exchange Rate
+-------------------------
+
+To create a new exchange rate you will need to call the ``/api/v1/exchange-rates/`` endpoint with the ``POST`` method.
+
+Definition
+^^^^^^^^^^
+
+.. code-block:: text
+
+    POST /api/v1/exchange-rates/
+
++----------------+----------------+--------------------------------------+
+| Parameter      | Parameter type | Description                          |
++================+================+======================================+
+| Authorization  | header         | Token received during authentication |
++----------------+----------------+--------------------------------------+
+| ratio          | request        | Ratio of the Exchange Rate           |
++----------------+----------------+--------------------------------------+
+| sourceCurrency | request        | The source currency                  |
++----------------+----------------+--------------------------------------+
+| targetCurrency | request        | The target currency                  |
++----------------+----------------+--------------------------------------+
+
+Example
+^^^^^^^
+
+.. code-block:: bash
+
+    $ curl http://demo.sylius.orgg/api/v1/exchange-rates/ \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Content-Type: application/json" \
+        -X POST \
+        --data '
+            {
+                "ratio": "0,8515706",
+                "sourceCurrency": "EUR",
+                "targetCurrency": "GBP"
+            }
+        '
+
+.. tip::
+
+    Remember, that before you will be able to add a new exchange rate both currencies have to be already defined.
+
+Exemplary Response
+^^^^^^^^^^^^^^^^^^
+
+.. code-block:: text
+
+    STATUS: 201 CREATED
+
+.. code-block:: json
+
+    {
+        "id":1,
+        "ratio":0.85157,
+        "sourceCurrency":{
+            "id":2,
+            "code":"EUR",
+            "createdAt":"2017-02-23T14:57:53+0100",
+            "updatedAt":"2017-02-23T14:57:53+0100",
+            "_links":{
+                "self":{
+                    "href":"\/api\/v1\/currencies\/EUR"
+                }
+            }
+        },
+        "targetCurrency":{
+            "id":3,
+            "code":"GBP",
+            "createdAt":"2017-02-23T14:58:20+0100",
+            "updatedAt":"2017-02-23T14:58:20+0100",
```

Comment:
I think that `createdAt` and `updatedAt` fields are not really useful for the Currency entity and we should not serialize them.